.azure-pipelines.yml

```diff
@@ -0,0 +1,206 @@
+      vmImage: $(imageName)
+    steps:
+    - task: UsePythonVersion@0
+      displayName: Enable Python 3.6
+      inputs:
+        versionSpec: '3.6'
+        addToPath: true
+        architecture: 'x64'
+
+    # Linux requirements.
+    - bash: |
+        # Install apt packages.
+        sudo apt-get -qq install cvs
+
+        # Uninstall mono-devel because it contains the "lc" command which conflicts with BoyterLc.
+        sudo apt-get -qq remove mono-devel
+
+        # Install NPM packages.
+        sudo npm install -g bower@$BOWER_VERSION
+
+        # Install Python packages.
+        export PATH=$PATH:~/.local/bin
+        pip install --user \
+          conan==$CONAN_VERSION \
+          pipenv==$PYTHON_PIPENV_VERSION \
+          reuse \
+          virtualenv==$VIRTUALENV_VERSION
+        conan user # Create the conan data directory. Automatic detection of your arch, compiler, etc.
+
+        # Install Ruby packages.
+        sudo gem install bundler -v $BUNDLER_VERSION
+
+        # Downgrade Rust, because the CargoSubcrateTest fails with the pre-installed version.
+        rustup default $RUST_VERSION
+
+        # Install Flutter.
+        curl -Os https://storage.googleapis.com/flutter_infra/releases/stable/linux/flutter_linux_$FLUTTER_VERSION.tar.xz
+        tar xf flutter_linux_$FLUTTER_VERSION.tar.xz -C $(dirname $FLUTTER_HOME)
+        rm flutter_linux_$FLUTTER_VERSION.tar.xz
+        export PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin"
+        flutter config --no-analytics
+        flutter doctor
+
+        # Install git-repo.
+        mkdir ~/bin
+        export PATH=$PATH:~/bin
+        curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
+        chmod a+x ~/bin/repo
+
+        # Install Go Dep.
+        mkdir -p ~/go/bin
+        export PATH=$PATH:~/go/bin
+        curl https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh
+
+        # Update PATH for next steps.
+        echo "##vso[task.setvariable variable=path;]$PATH"
+      condition: eq( variables['Agent.OS'], 'Linux' )
+      displayName: Install Linux requirements
+
+    # TODO: Add a step to run "reuse lint" (only on Linux?).
+
+    # Windows requirements.
+    - pwsh: |
+        # Install Chocolatey packages.
+        cinst dep --version $env:GO_DEP_VERSION -y --no-progress
+        cinst haskell-stack --version $env:STACK_VERSION -y --no-progress
+        refreshenv
+
+        # Install CVS. Disabled because msys2 installation takes too long.
+        #cinst msys2 --params "/InstallDir=C:/msys64" --no-progress
+        #C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy cvs"
+        #$env:PATH += ";C:\msys64\usr\bin"
+
+        # Install NPM packages.
+        npm install -g bower@$env:BOWER_VERSION
+
+        # Install Python packages.
+        pip install --user conan==$env:CONAN_VERSION pipenv==$env:PYTHON_PIPENV_VERSION reuse virtualenv==$env:VIRTUALENV_VERSION
+        pip show conan
```

Comment:
What does this do? Is this left-over debug code?

Reply:
Yes, removed.